src/serverSide.ts

```diff
@@ -0,0 +1,32 @@
+import { zencode_exec } from "zenroom";
+import { readStringFromFile } from "./service/fileService";
+import * as dotenv from "dotenv";
+const DEFAULT_SERVER_SIDE_CONTRACT =
+  "./node_modules/keypair-lib/lib/zencode/Keypair-Creation-Server-Side.zen";
```

Comment:
Are these './node_modules/keypair-lib' path okay? If  I try to `yarn build && yarn test` on my local machine the tests fails

Reply:
please test it with an .env file. Path are ok but I will replace it with a path using this: https://nodejs.org/api/modules.html#modules_dirname that I just discovered yesterday

Reply:
btw new version doesn't have node_modules anymore